cmd/templates.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+func getUsageTemplate() string {
+	return `Usage:
+  {{.CommandPath}} [command]
+
+Commands:
+  {{rpad "setup" 29}} Setup guide to create your first feature flag
+  {{rpad "config" 29}} View and modify specific configuration values
+  {{rpad "completion" 29}} Generate the autocompletion script for the specified shell
+
+Resource Commands:
+  {{rpad "flags" 29}} List, create, modify, and delete feature flags, their statuses, and their expiring targets
+  {{rpad "environments" 29}} Create, delete, and update environments
+  {{rpad "projects" 29}} Create, destroy, and manage projects
+  {{rpad "members" 29}} Invite new members to an account
+  {{rpad "segments" 29}} List, create, modify, and delete segments
```

Comment:
these are snippets pulled out from the tag description in the openapi spec for each of these resources

Reply:
Well, this is embarassingly inconsistent 🙈 sorry about that! I can get this updated in the spec. In the meantime, couple of suggestions:

```suggestion
  {{rpad "flags" 29}} List, create, and modify feature flags and their targeting
  {{rpad "environments" 29}} List, create, and manage environments
  {{rpad "projects" 29}} List, create, and manage projects
  {{rpad "members" 29}} Invite new members to an account
  {{rpad "segments" 29}} List, create, modify, and delete segments
```
reasoning: 
- flags: flag status is very de-emphasized in new shell; you don't usually delete them (usually you archive); "modify" (the semantic patch) has most of its semantic patch instructions around variations and targeting
- environments, projects: use "manage" because you typically don't delete these either (and also so that not every description starts "list, create, modify, and delete" -- will be harder to read)